### PR TITLE
Add restart support for DRSDT

### PR DESCRIPTION
**Pull request by vkip**
Only DRSDT, not DRSDTR for now.

### CODE REVIEW

File: opm/io/eclipse/rst/state.cpp [hunk 1]
```diff
@@ -90,10 +90,23 @@ RstState::RstState(std::shared_ptr<EclIO::RestartFileView> rstView,
...
 
+void RstState::load_oil_vaporization(const std::vector<int>& intehead,
+                                     const std::vector<double>& doubhead)
+{
+    const std::size_t numPvtRegions = this->oilvap.numPvtRegions();
+    std::vector<double> maximums(numPvtRegions);
+    std::vector<std::string> options(numPvtRegions);
+    const auto tconv = this->unit_system.to_si(::Opm::UnitSystem::measure::time, 1.0);
+    std::fill(maximums.begin(), maximums.end(), doubhead[VI::doubhead::dRsDt]/tconv);
+    std::fill(options.begin(), options.end(), intehead[VI::intehead::DRSDT_FREE]==1 ? "FREE" : "ALL");
```
Comment (bska) [inline opm/io/eclipse/rst/state.cpp:107]: Tiny optimisation: You don't need construct+fill here, you can form both `maximums` and `options` in a single statement (each):
```C++
std::vector<double> maximums(numPvtRegions, doubhead[VI::doubhead::dRsDt]/tconv);
std::vector<std::string> options(numPvtRegions, intehead[VI::intehead::DRSDT_FREE]==1 ? "FREE" : "ALL");
```
It probably does not make any significant difference, but if you'd like to change this then please go ahead.

**Review by bska — APPROVED** [review]
This looks good to me and I'm prepared to merge this into the master branch now.  There is one (tiny) performance opportunity here that you may or may not wish to exploit.

File: opm/io/eclipse/rst/state.cpp (hunk 1 above)
Comment (vkip) [reply opm/io/eclipse/rst/state.cpp]: Thanks, good point. Updated now.

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**